### PR TITLE
[DOCS] Readme changes

### PR DESCRIPTION
**Pull request by tsavina**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Review by kblaszczak-intel — APPROVED** [review]

**Review by rkazants — COMMENTED** [review]
Let us have a different structure. Not sure that we need table of content in the main page.
1.	Install OpenVINO
2.	Deploy your model
     a.	Here we will show snippets to run PyTorch, TensorFlow, ONNX models using OpenVINO (we have such snippets in user docs) – need animations
     b.	Show how easy to change HW to infer and configure asynchronous calls – need animations
      c.	Other APIs (C/C++, JS)
3.	Examples and Tutorials
     a.	Links to most populat notebooks samples:
     b.	Link 1
      c.	Link 2
     d.	Link 3
4.	User documentation
     a.	Just couple of sentences and link to the user documentation
5.	Developer documentation
     a.	Just one sentence and link to documentation
6.	Community
     a.	Link to a doc with pro-active external contribotors
     b.	Link to Discord

Good example of main page: https://github.com/Lightning-AI/pytorch-lightning

**Comment by rkazants** [comment]
Let developers take a look before merge.

Best regards,
Roman

**Comment by mlukasze** [comment]
I would like to add information about [GenAI](https://github.com/openvinotoolkit/openvino.genai) and [tokenizers](https://github.com/openvinotoolkit/openvino_tokenizers), but I have to clue in what section could be a best place for them. 
What is your opinion?

**Comment by mlukasze** [comment]
also, we have a conflict ;)

File: README.md
Comment (tsavina) [inline README.md:83]: @moslex can you help with this section?

**Comment by tsavina** [comment]
> I would like to add information about [GenAI](https://github.com/openvinotoolkit/openvino.genai) and [tokenizers](https://github.com/openvinotoolkit/openvino_tokenizers), but I have no clue what section could be a best place for them. What is your opinion?

added it to the OV Ecosystem section for now

File: README.md
Comment (mlukasze) [reply README.md:84]: let me block merge until decision on this section will be made, ping me when discussion will be solved, please

File: README.md
Comment (mlukasze) [inline README.md:69]: Tools in this section are technically "powered by OV", but maybe we could consider to highlight them as part of official "Development toolkit", and move to dedicated section? 
@moslex @andrei-kochin @MaximProshin calling for your opinion here :)

**Review by mlukasze — CHANGES_REQUESTED** [review]

File: README.md
Comment (rkazants) [inline README.md:46]: Propose to remove a section `Components`. this is mostly for developers. For developer we should have a link here to developer documentation. And it should be enough

File: README.md
Comment (rkazants) [inline README.md:21]: Propose to remove `Contents` it wastes a space

File: README.md
Comment (rkazants) [inline README.md:138]: please remove License. It duplicates what we have on the right side of the OpenVINO Github page.

File: README.md
Comment (rkazants) [inline README.md:55]: Let us move `Installation` to the very top. And no need to have a link to instruction. It wastes more space than if we explicitly share instruction how to install:

```
pip install openvino
```

**Review by rkazants — CHANGES_REQUESTED** [review]

File: README.md
Comment (mlukasze) [reply README.md:138]: is it acceptable by legal?

File: README.md
Comment (mlukasze) [reply README.md:55]: Agree, documentation should reflect natural flow of software usage - first you have to install, then - start using

File: README.md
Comment (rkazants) [reply README.md:138]: Okay, let leave it. I just checked PyTorch and TensorFlow. And they have it. PyTorch Lightning - not.
Let us put license section to the very bottom

File: README.md
Comment (rkazants) [reply README.md:46]: Developer documentation should contain a list of components and more detail info

File: README.md
Comment (moslex) [reply README.md:61]: I would recommend to have this link only instead of listing products which use OpenVINO:
A curated list of OpenVINO based AI projects: https://github.com/openvinotoolkit/awesome-openvino

File: README.md
Comment (moslex) [reply README.md:46]: If you think that ecosystem word is confusing here, we can state it as OpenVINO workflow projects 
image
We can add openvino_tokenizers as well here in the list https://github.com/openvinotoolkit/openvino_tokenizers.
May be we can even add this picture or link to it.

**Comment by tsavina** [comment]
@rkazants @mlukasze @moslex please take a look

**Review by mlukasze — APPROVED** [review]

File: README.md
Comment (rkazants) [inline README.md:73]:
```suggestion
```
Let us remove mention of Release Notes because we have to update every release and I see no value to have it here. RN should be bound to exact release package.

Comment (rkazants) [reply README.md:73]: I see this section as additional educational resources about OpenVINO

File: README.md
Comment (rkazants) [inline README.md:76]:
```suggestion
```
I see this section as additional educational resources about OpenVINO

File: README.md
Comment (rkazants) [inline README.md:51]:
```suggestion
[**User documentation**](https://docs.openvino.ai/) contains detailed information about how to optimize and deploy your model for inference using OpenVINO.
```

File: README.md
Comment (rkazants) [inline README.md:53]:
```suggestion
```

File: README.md
Comment (rkazants) [inline README.md:55]:
```suggestion
[**Developer documentation**](./docs/dev/index.md) focuses on how OpenVINO [components](./docs/dev/index.md#openvino-components) work and describes [building](./docs/dev/build.md) and [contributing](./CONTRIBUTING.md) processes.
```

File: README.md
Comment (rkazants) [inline README.md:57]:
```suggestion
```

File: README.md
Comment (tsavina) [reply README.md:32]: could you please provide these code examples? I'm not sure which models and code snippets we want to use here.

File: README.md
Comment (tsavina) [reply README.md:73]: Release Notes page in docs will contain all changes throughout 2024 and can be used to track changes and see what's new, I think we could keep a link to it

File: README.md
Comment (tsavina) [reply README.md:76]: sure, if it's not required (question to @moslex), let's remove this link

File: README.md
Comment (rkazants) [reply README.md:73]: It contains only features for 2024.0 release and we have to update it every release. Not sure it gives much values for regular users. People who really wants to see new features will walk the standard way that means go to releases tab and find release notes for required release.

File: README.md
Comment (rkazants) [reply README.md:32]: @mvafin, we need to provide some valuable examples on the main page of GitHub to demonstrate how to infer PyTorch and TensorFlow models using OV. Or even how to migrate from TF to OV and from PT for OV.

File: README.md
Comment (moslex) [reply README.md:76]: > sure, if it's not required (question to @moslex), let's remove this link

Ok to remove

File: docs/dev/index.md
Comment (moslex) [inline docs/dev/index.md:21]: Please replace Model Optimizer with Model Converter

File: README.md
Comment (rkazants) [inline README.md:20]: let us use links with `latest` like `https://docs.openvino.ai/latest/_static/download/OpenVINO_Quick_Start_Guide.pdf`. Otherwise, we will have to update is again and again.
The same comment relates other links

File: README.md
Comment (rkazants) [inline README.md:87]:
```suggestion
OpenVINO also supports CPU, GPU, and NPU devices and works with models in TensorFlow, PyTorch, ONNX, TensorFlow Lite, PaddlePaddle model formats.
```

File: README.md
Comment (rkazants) [inline README.md:15]: Should we mention LLMs, GenAI models here?
@moslex, @andrei-kochin, please advice

File: README.md
Comment (rkazants) [inline README.md:93]:
```suggestion
-   [🤗Optimum Intel](https://github.com/huggingface/optimum-intel) -  a simple interface to optimize Transformers and Diffusers models.
```

**Review by rkazants — COMMENTED** [review]
Left last comments. Other looks GREAT

**Review by rkazants — APPROVED** [review]

File: README.md
Comment (rkazants) [inline README.md:67]:
```suggestion
```python
```

File: README.md
Comment (rkazants) [inline README.md:45]:
```suggestion
```python
```

File: README.md
Comment (rkazants) [inline README.md:26]:
```suggestion
```sh
```

File: README.md
Comment (rkazants) [inline README.md:15]: Should we also have a separate bullet to mention different APIs: Python, C/C++, and JS?
@mlukasze, @ilya-lavrenov

File: README.md
Comment (rkazants) [reply README.md:20]: ok for now, but `latest` is definetely better

**Comment by rkazants** [comment]
@tsavina, let us merge this one and address last comments in other PR if needed

**Comment by rkazants** [comment]
@tsavina, any update on this PR? Let us merge and continue improvement in separate PRs

**Comment by rkazants** [comment]
@moslex, @ilya-lavrenov, please take a look.

Best regards,
Roman

File: README.md
Comment (ilya-lavrenov) [inline README.md:41]: can we put PyTorch as first FW?

File: README.md
Comment (ilya-lavrenov) [inline README.md:60]: can  we use callable instead of `infer_new_request` ?

File: README.md
Comment (ilya-lavrenov) [inline README.md:82]: callable

File: README.md
Comment (moslex) [reply README.md:13]: Yes, I would add "large and small language models, generative AI models"

File: CONTRIBUTING.md
Comment (moslex) [inline CONTRIBUTING.md:96]: - [Model Optimizer](https://github.com/openvinotoolkit/openvino/tree/master/tools/mo) - Should be replaced with OCV

File: docs/dev/index.md
Comment (moslex) [inline docs/dev/index.md:55]: NPU is missed

**Review by moslex — APPROVED** [review]
Ok to commit. Please, address my comments in the next round of changes.

File: README.md
Comment (rkazants) [reply README.md]: I will address it in separate PR

File: README.md
Comment (rkazants) [reply README.md]: I will address it in separate PR

**Comment by popovaan** [comment]
Why OpenVino Telemetry information is removed?

**Pull request merged**
